deploy/roles/r-update/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: Install packages required by R packages
+  become: yes
+  become_method: sudo
+  yum:
+    name: "{{ item }}"
+    state: present
+    disable_gpg_check: yes
+  with_items: "{{ r_dependencies }}"
+  tags:
+    - r-update
```

Comment:
This section should probably be in `r-libs`, since there may be times that that role is run without updating first, but new rpm dependencies were introduced.  It might be easier to remove the separate role for `r-update`, and includ the update tasks in `r-libs`, and go back to using the `when: update_r_packages` conditional, or you could just move the `include_role` task to after this task is executed.

Reply:
@lel99999 It seems like you made this change in r-libs, but did not remove r-update, can you remove the r-update files since they are no longer needed?